examples/python/pincell_depletion/restart_depletion.py

```diff
@@ -0,0 +1,84 @@
+import openmc
+import openmc.deplete
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+#                      Simulation Input File Parameters
+###############################################################################
+
+# OpenMC simulation parameters
+batches = 100
+inactive = 10
+particles = 1000
+
+# Depletion simulation parameters
+time_step = 1*24*60*60 # s
+final_time = 5*24*60*60 # s
+time_steps = np.full(final_time // time_step, time_step)
+
+chain_file = './chain_simple.xml'
+power = 174 # W/cm, for 2D simulations only (use W for 3D)
+
+###############################################################################
+#                      Load previous simulation results
+###############################################################################
+
+# Load geometry from statepoint
+statepoint = 'statepoint.100.h5'
+sp = openmc.StatePoint(statepoint)
+geometry = sp.summary.geometry
+
+# Close statepoint and summary files to be able to write over them
+sp.close()
```

Comment:
The `StatePoint` class can be used as a context manager in Python, so you can change this to:
```Python
with openmc.StatePoint(statepoint) as sp:
    geometry = sp.summary.geometry
```
And then you don't need to call `sp.close()` at the end (it happens automagically)